package.json

```
@@ -11,15 +11,18 @@
   "devDependencies": {
     "@changesets/cli": "^2.29.2",
     "@devcontainers/cli": "^0.75.0",
+    "@vitest/coverage-v8": "^3.2.4",
     "eslint": "8.57.0",
     "prettier": "3.2.5",
     "turbo": "^2.5.2",
-    "typescript": "~5.8.3"
+    "typescript": "~5.8.3",
+    "vitest": "^3.2.4"
   },
   "scripts": {
     "build": "turbo run build",
     "code-review": "turbo run typecheck format:check lint:check",
```

I suggest to call the root-level `test:coverage` script during `code-review`

```suggestion
    "code-review": "turbo run typecheck format:check lint:check //#test:coverage",
```